scripts/apps/search/components/fields/scheduledDateTime.tsx

```diff
@@ -16,7 +17,7 @@ class ScheduledDateTime extends React.PureComponent<IProps> {
     render() {
         const scheduled = this.props.item.archive_item
             ? this.props.item.archive_item.schedule_settings.utc_publish_schedule
-            : appConfig?.features?.showPublishSchedule ? this.props.item?.publish_schedule : null;
+            : this.props.item?.publish_schedule || null;
```

Comment:
still not sure it's ok here, `utc_publish_schedule` is in utc, but `publish_schedule` should be local time, so those are different, can you explain how that works pls?

Reply:
okay, I got your point, Actually, we are not getting `schedule_settings` for the `in_progress` or `submitted` items, so that's why I used `publish_schedule` but I think we need `utc_publish_schedule` for `in_progress` or `submitted` items as well, am I right?